src/tests/EdFi.DataManagementService.Tests.E2E/Features/SuperclassReferenceValidation.feature.disabled

```diff
@@ -0,0 +1,120 @@
+            Given educationOrganizationId 102
+             When a POST request is made to "ed-fi/programs" with
+                  """
+                  {
+                      "programName": "Program Non-Existing Test",
+                      "programTypeDescriptor": "uri://ed-fi.org/ProgramTypeDescriptor#Bilingual",
+                      "educationOrganizationReference": {
+                          "educationOrganizationId": 102
+                      }
+                  }
+                  """
+             Then it should respond with 409
+              And the response body is
+                  """
+                  {
+                      "detail": "The referenced 'educationOrganizationId' resource does not exist.",
+                      "type": "urn:ed-fi:api:conflict:invalid-reference",
+                      "title": "Resource Not Unique Conflict due to invalid-reference",
+                      "status": 409,
+                      "correlationId": null
+                  }
+                  """
+        @ignore
+        Scenario: Ensure clients can update a program that references to an existing school so it references now the Local Education Agency
+            Given programName Program School Test
+            #set value to {id}
+             When a PUT request is made to "ed-fi/programs/{id}" with
+                  """
+                  {
+                      "programName": "Program School Test",
+                      "programTypeDescriptor": "uri://ed-fi.org/ProgramTypeDescriptor#Bilingual",
+                      "educationOrganizationReference": {
+                          "educationOrganizationId": 101
+                      }
+                  }
+                  """
+             Then it should respond with 204
+
+        @ignore
+        Scenario: Ensure clients cannot update a program that references to an existing Local Agency so it references now a Non Existing Education Organization
+            Given programName Program LEA Test
+             #set value to {id}
+             When a PUT request is made to "ed-fi/programs/{id}" with
+                  """
+                  {
+                      "programName": "Program LEA Test",
+                      "programTypeDescriptor": "uri://ed-fi.org/ProgramTypeDescriptor#Bilingual",
+                      "educationOrganizationReference": {
+                          "educationOrganizationId": 102
+                      }
+                  }
+                  """
+             Then it should respond with 409
+                  """
+                  {
+                      "detail": "The referenced 'educationOrganizationId' resource does not exist.",
+                      "type": "urn:ed-fi:api:conflict:invalid-reference",
+                      "title": "Resource Not Unique Conflict due to invalid-reference",
+                      "status": 409,
+                      "correlationId": null
+                  }
```

Comment:
Here is the error message when posting a StudentProgramAssociation in the ODS/API 7.2, and the Program does not exist:

```
{
  "detail": "The referenced 'Program' item does not exist.",
  "type": "urn:ed-fi:api:data-conflict:unresolved-reference",
  "title": "Unresolved Reference",
  "status": 409,
  "correlationId": "31cfd749-5c91-45bf-b66d-469dcf0dd377"
}
```

There are some important differences between this and what you have above. But what you have above must have been inspired by something real. I'm looking into this a little more and will edit my comment after more investigation.

Reply:
Changed